admin/tools/docker/base/Dockerfile

```diff
@@ -50,22 +50,20 @@ RUN dnf install -y 'dnf-command(config-manager)' \
         patch \
         protobuf-compiler \
         protobuf-devel \
-        python38-devel \
+        python3-devel \
         tree \
         vim \
     && dnf clean all \
     && rm -rf /var/cache/yum
 
-ENV ARROW_VERSION 11.0.0
-
 RUN dnf update -y \
     && dnf install -y \
        https://apache.jfrog.io/artifactory/arrow/almalinux/$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1)/apache-arrow-release-latest.rpm \
     && dnf config-manager --set-enabled epel \
-    && dnf config-manager --set-enabled powertools \
+    && dnf config-manager --set-enabled crb \
     && dnf install -y \
-      arrow-devel-$ARROW_VERSION \
-      parquet-devel-$ARROW_VERSION \
+      arrow-devel \
```

Comment:
You explicitly call out versions 13 below -- should these be explicit to match?

Reply:
The older version of the run container had these lines where the version was mentioned twice (in different form):
```
ENV ARROW_VERSION 11.0.0

dnf install -y arrow11-libs-$ARROW_VERSION \
dnf install -y parquet11-libs-$ARROW_VERSION \
```
And the build container in the older version had:
```
dnf install -y arrow-devel-$ARROW_VERSION
```
You may see the "assymetry". I have no clue why developers chose this naming convention. The only theory I have is that the packages are named after the major version, while leaving some degree of freedom for the middle an dminor version numbers.

AlmaLinux 9 comes with version 13.0.0 of the tools. So, we don't need to mention it in the build container. The run container still requires:
```
dnf install -y arrow13-libs
dnf install -y parquet13-libs
```